phpunit.xml.dist

```diff
@@ -36,6 +36,7 @@
 
         <env name="APP_ENV" value="test"/>
         <env name="SHELL_VERBOSITY" value="-1" />
+        <env name="TPAY_API_URL" value="http://localhost:4000" />
```

Comment:
I assume it will create a default value if it is not set in env? 
Will note it down for documentation 📓

Reply:
It's empty, but it does not do anything; this value is only for testing purposes to point TPAY API to Mockoon. In every other scenario it should not be set.